src/mlpack/methods/ann/init_rules/glorot_init.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file glorot_init.hpp
+ * @author Prabhat Sharma
+ *
+ * Definition and implementation of the Glorot initialization method. This
+ * initialization rule initialize the weights to maintain activation variances
+ * and back-propagated gradients variance as one moves up or down the network.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @inproceedings{pmlr-v9-glorot10a,
+ * title = {Understanding the difficulty of training deep feedforward neural networks},
+ * author = {Xavier Glorot and Yoshua Bengio},
```

Comment:
Maybe you can remove whitespaces all together on sides of `=` as most of the codebase follows the same design.